thundra/plugins/log/log_plugin.py

```diff
@@ -10,30 +13,73 @@ def __init__(self):
             'after:invocation': self.after_invocation
         }
         self.log_data = {}
+        self.tracer = ThundraTracer.getInstance()
+        self.scope = None
 
     def before_invocation(self, data):
         context = data['context']
         logs.clear()
+        function_name = getattr(context, constants.CONTEXT_FUNCTION_NAME, None)
+        active_span = self.tracer.get_active_span()
         self.log_data = {
-            'transactionId': data['transactionId'],
-            'applicationName': getattr(context, 'function_name', None),
+            'type': "Log",
+            'agentVersion': '',
```

Comment:
Even though this can't be detected automatically, we can get this from constants and the constant is updated manually for every release. Can start with `2.0.0` for the first release with new data model